polysemy-zoo.cabal

```diff
@@ -28,6 +28,8 @@ source-repository head
 library
   exposed-modules:
       Polysemy.KVStore
+      Polysemy.Random
```

Comment:
I'd prefer if we move this in a separate PR

Reply:
Sure.  I'll take it out.  I thought it might facilitate the conversation about trying to unify them

Reply:
Gotcha. I'm not sure it's worth spending time on --- we can let someone else figure out how to put them together if they care enough.